src/main/java/namor/template/calculator/RPNcalculator.java

```diff
@@ -0,0 +1,65 @@
+package namor.template.calculator;
+
+import namor.template.stack.ArrayStack;
+
+public class RPNcalculator {
+    private ArrayStack numbers = new ArrayStack();
```

Comment:
You need this to encapsulate your stack in such a way, which will help you to avoid situation when some method change something in stack and everything crashes.